src/rust/catmem/mod.rs

```diff
@@ -131,6 +146,8 @@ impl CatmemLibOS {
         qd: QDesc,
         yielder: Yielder,
     ) -> (QDesc, OperationResult) {
+        #[cfg(feature = "profiler")]
+        timer!("catmem::close_coroutine");
```

Comment:
No need to benchmark the async co-routine as well, as it may take time to finish up.

Reply:
Removed in 6c6f6b77268c73907501add92ea403fcea4eb0e6.